features/import_export/export_command.feature

```diff
@@ -0,0 +1,35 @@
+@import_export
+Feature:
+    In order to export data from my store
+    As a store owner
+    I want to be able to run jobs based on defined export profiles
+
+    Background:
+        Given there are following export profiles configured:
+          | name               | description | code        | reader   | reader_configuration                  | writer     | writer_configuration                         |
+          | UsersExportProfile | Lorem ipsum | user_export | user_orm | batch_size:10,date_format:Y-m-d H:i:s | csv_writer | Delimiter:;,Enclosure:",File:/tmp/output.csv |
+        And there are following users:
+          | email          | enabled  | created_at          |
```

Comment:
I'm sure that `enabled` and `created_at` (btw. should be `created at` with space in the middle instead of `_`) are not needed in this scenario and gives no business value.